### PR TITLE
fix: display datetime time value when disabled

### PR DESCRIPTION
**Pull request by Skaiir**
Related to https://github.com/camunda/camunda-modeler/issues/3383

### CODE REVIEW

File: packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ export default function Datepicker(props) {
   useEffect(() => {
 
     if (disabled) {
+      flatpickrInstance && flatpickrInstance.setDate(null, true);
```
Comment (pinussilvestrus) [inline packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: This shows the placeholder in any case when disabled, also when valid input data was given.

image

Bildschirmfoto 2023-01-12 um 13 13 47

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: This is inconsistent with other fields, where we display the initial value even on disabled.

image

Comment (Skaiir) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: Wait this confuses me a bit. Is the goal of 'disabled' to work like a read-only then?

Comment (Skaiir) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: I think a few components need adjustment then

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: It's not `readonly` as it isn't submitted. However, it should be displayed if data is imported (cf. https://bpmn.io/blog/posts/2022-form-js-default-values-improved-submit.html).

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: > I think a few components need adjustment then

Indeed it seems like this isn't working for the `taglist` as well.
image
image

I'll create an issue for that.

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: Not that big of a deal, as we should discourage the use of `disabled` fields I believe (cf. #362)

Comment (pinussilvestrus) [reply packages/form-js-viewer/src/render/components/form-fields/parts/Datepicker.js:54]: Here you go: https://github.com/bpmn-io/form-js/issues/496

**Comment by Skaiir** [comment]
We now have the expected behavior: 

image

**Review by vsgoulart — APPROVED** [review]

**Pull request merged**